CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.7.0-pre.5 (2025-06-16)
+_The below list is very much incomplete._
+
+### Changed
+- Replace `Limb::mac` with `::carrying_add_mul` ([#817]).
+  - Note: `::carrying_add_mul` requires a different parameter order than `::mac`.
```

Comment:
These should all be `carrying_mul_add`, right?

```suggestion
- Replace `Limb::mac` with `::carrying_mul_add` ([#817]).
  - Note: `::carrying_mul_add` requires a different parameter order than `::mac`.
```

We also noted this in the deprecation message. I guess that wasn't helpful. Can we make it more visible?